azure-pipelines.yaml

```diff
@@ -112,3 +121,17 @@ stages:
               pr_number: $(pull_request_number)
               merge_method: 'merge'
             displayName: Merge PR# $(System.PullRequest.PullRequestNumber)
+
+  - stage: PoliCheck
```

Comment:
For consistency with what I have for the designer repo, call the stage code_analysis and a display name of "Code Analysis" and move the stage to the top of the file.

Reply:
Done.